static/js/editor.js

```diff
@@ -73,10 +76,13 @@ $(document).ready(function() {
         if (menu === undefined) {
             menu = $(".ui.dropdown.examples .menu");
         }
+        console.log(menu);
```

Comment:
Can you remove these console logs, no need to pollute a user's browser with debugging lines